package.json

```diff
@@ -28,7 +28,7 @@
   ],
   "dependencies": {
     "async": "^1.2.1",
-    "pad-stdio": "^1.0.0"
+    "pad-stdio": "^2.0.0"
```

Comment:
I think this needs to be ^2.0.1 as it is the minimum version for this to work properly.